bin/yb-ctl

```diff
@@ -1090,6 +1114,12 @@ class ClusterControl:
                 "--ip_start", default=DEFAULT_IP_START, type=int,
                 help="Start index for IP address")
 
+            subparsers[cmd].add_argument(
+                "--listen_ip",
```

Comment:
`bind_ip` would probably be better ?

Reply:
Also, shouldn't this also be included in the `stop` subparsers? Does `yb-ctl stop` work if you have `yb-ctl stop --listen_addr 11.11.11.11`?

Reply:
stop works fine without the listen_addr because it searches for procs based on rpc_bind_address and I didn't touch that.